### ISSUE TITLE
Odd mouse wheel zoom step of 0.24

### ISSUE THREAD

**Issue by Nekzuris** [facebook/Rapid#1651]
### Description

This probably doesn't bother anyone, but if you start to zoom out from z18 you end up at z17.04 because of the 0.24 zoom step.
It would be much nicer if it were 0.25 or 0.20 like iD to stay on round zoom levels for layer with text like Mapnik.

### Screenshots

z17.00
![z17 00](https://github.com/user-attachments/assets/aea9e3bc-8fd5-4caf-afdb-c6729e2c0cab)
z17.04
![z17 04](https://github.com/user-attachments/assets/1d4a67ad-7395-4f7e-9844-b72ae362d06d)

### Version

2.5.0-pre.0

### What browser are you seeing the problem on? What version are you running?

Firefox v133.0

### The OS you're using

win

### Steps to reproduce

1. Open the editor at z18
2. Zoom out 4 steps with the mouse wheel

### The browser URL at the time you encountered the bug

https://rapideditor.org/canary#map=18.00/48.27186/7.05606&background=MAPNIK&datasets=fbRoads,msBuildings&disable_features=boundaries

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0

**Comment by bhousel** [comment]
I don't think these values are really standard across browsers or mice.. 
The user can also change their page zoom and we'd see different values from the wheel event.

So for example, I have a Logitech MX and I'm using MacOS:

In Chrome 133:
* open editor at z18
* 1 click: 18.024007464843752
* 2 clicks: 18.0480149296875

In Firefox 130:
* open editor at z18
* 1 click:  18.096
* 2 clicks:  18.191999999999997


I also tried this in iD and got other different numbers:
In Chrome 133:
* open editor at z18
* 1 click: 18.00800048828125
* 2 clicks: 18.0160009765625

In Firefox 130:
* open editor at z18
* 1 click:  18.032000000000004
* 2 clicks:  18.064

The tldr is - Rapid will receive values from the browser when the user scrolls, but these aren't the sort of thing that translate neatly to "wheel clicks".  We're really aiming more for "feels right".

**Comment by Nekzuris** [comment]
I guess we could listen to [wheel event](https://developer.mozilla.org/en-US/docs/Web/API/Element/wheel_event) and not use the delta, but simply consider each event as a step, but yeah it might not feels right for every mouse or trackpad.
